.travis.yml

```diff
@@ -43,4 +43,4 @@ install:
     - conda install -n root --yes --quiet jinja2 anaconda-client
 
 script:
-    - conda-build-all ./ --inspect-channels scitools/label/${LABEL_NAME} ${UPLOAD_CHANNELS} --matrix-condition "numpy >=1.8,,<=1.10" "python >=2.7,<3|>=3.4,<3.5|>=3.5,<3.6"
+    - conda-build-all ./ --inspect-channels scitools/label/${LABEL_NAME} ${UPLOAD_CHANNELS} --matrix-condition "numpy >=1.8,,<=1.11" "python >=2.7,<3|>=3.4,<3.5|>=3.5,<3.6"
```

Comment:
I think the extra comma in the numpy condition
`numpy >=1.8,,<=1.11` rather than `numpy >=1.8,<=1.11`
 is for some reason stopping the travis build from building against numpy 1.11